src/components/screens/SendScreen/screens/TransactionAmountScreen.tsx

```diff
@@ -83,31 +138,80 @@ const TransactionAmountScreen: React.FC<TransactionAmountScreenProps> = ({
     handlePercentagePress,
   } = useTokenFiatConverter({ selectedBalance });
 
+  const spendableBalance = useMemo(() => {
+    if (!selectedBalance) return new BigNumber(0);
+
+    if (
+      selectedBalance.assetType !== AssetTypeWithCustomToken.NATIVE &&
+      selectedBalance.assetType !== AssetTypeWithCustomToken.CREDIT_ALPHANUM4 &&
+      selectedBalance.assetType !==
+        AssetTypeWithCustomToken.CREDIT_ALPHANUM12 &&
+      selectedBalance.assetType !== AssetTypeWithCustomToken.CUSTOM_TOKEN
+    ) {
+      return new BigNumber(selectedBalance.total);
+    }
+
+    if (selectedBalance.assetType !== AssetTypeWithCustomToken.NATIVE) {
+      return new BigNumber(selectedBalance.total);
+    }
+
+    const currentBalance = new BigNumber(selectedBalance.total);
+    const feeStroops = new BigNumber(transactionFee).multipliedBy(1e7);
+    const minBalance = new BigNumber(2 + subentryCount).multipliedBy(
+      BASE_RESERVE,
+    );
+    const calculatedSpendable = currentBalance
+      .minus(minBalance)
+      .minus(stroopToXlm(feeStroops));
```

Comment:
does it make some difference to convert the fee to stroops on line 159 and then convert it back to XLM? it's looking a bit redundant